src/widget/button.rs

```diff
@@ -55,16 +55,18 @@ impl Label {
         Label { text: text.into() }
     }
 
-    fn get_layout(&self, rt: &mut Piet, font_size: f64) -> <Piet as RenderContext>::TextLayout {
+    fn get_layout(
+        &self,
+        rt: &mut <Piet as RenderContext>::Text,
```

Comment:
when #113 lands, we'll be able to use `PietText` here (and `PietTextLayout`) below.

Reply:
(that's merged, so if you rebase this now you can use the type aliases)

Reply:
Okay :slightly_smiling_face: Maybe it's also a good time to move Label and DynLabel into their own file.

Reply:
How about we get this change merged, and then you can open a separate PR that adds the new file?

Reply:
Sure :slightly_smiling_face: